Open Judge System/Web/OJS.Web/packages.config

```diff
@@ -25,7 +25,7 @@
   <package id="Microsoft.AspNet.Web.Optimization" version="1.1.3" targetFramework="net45" />
   <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net45" />
   <package id="Microsoft.AspNet.WebPages" version="3.2.3" targetFramework="net45" />
-  <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="1.0.7" targetFramework="net45" />
+  <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="1.0.7" targetFramework="net45" requireReinstallation="true" />

```

Comment:
Revert this if it's not clear what the attribute will do.

Reply:
Done in https://github.com/SoftUni-Internal/OpenJudgeSystem/pull/222